### PR TITLE
Add Sequential requestor api domains

### PR DESCRIPTION
**Pull request by qroll**
**Changes**

- Add api domains for Sequential requestor endpoints, using `class-validator` and `class-transformer`
- Include additional setup for runtime decorators
- Note: webhook typings will be in a separate PR
-   [delete] branch

**Changelog entry**

-   Add Sequential requestor api domains

### CODE REVIEW

**Comment by qroll** [comment]
also requesting to update the repository settings to require approval before merging 👀

**Review by weili-govtech — APPROVED** [review]

**Comment by weili-govtech** [comment]
> also requesting to update the repository settings to require approval before merging 👀

cc: @keithtxw

**Pull request merged**
